aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -108,7 +108,13 @@ class PGCluster:
     _pg_roles: Dict[str, PGRole]
     _mangle: bool
 
-    def __init__(self, conn_info: Union[str, Dict[str, Any]], filtered_db: Optional[str] = None, mangle: bool = False):
+    def __init__(
+        self,
+        conn_info: Union[str, Dict[str, Any]],
+        filtered_db: Optional[str] = None,
+        filtered_roles: Tuple[str] = ((), ),
```

Comment:
The type for ```filtered_roles``` is wrong (tuple has fixed size, you need sequence or list or set) and the default argument type doesn't match.